scripts/withdrawMoney.py

```diff
@@ -1,33 +1,54 @@
 # Bailee Segars
 import pandas as pd
+from decimal import *
```

Comment:
```suggestion
from decimal import Decimal
```
import only what we need from module